test/multi-node-data/node0/sgnd/config/genesis.json

```diff
@@ -200,4 +198,4 @@
     },
     "upgrade": {}
   }
-}
\ No newline at end of file
+}
```

Comment:
hmm, why the format the field sequence of this multinode genesis remain the same, while the diff is so much for `test/data/.sgnd/config/genesis.json`? Or is this because multinode tests have not been updated and tested yet?